.travis.yml

```diff
@@ -4,11 +4,12 @@ sudo: false
 
 python:
   - '2.7'
-  - '3.4'
   - '3.5'
+  - '3.6'
 
 install:
   - pip install flake8
+  - pip install -r requirements.txt
```

Comment:
You might also want to test:  `pip install .` before the requirements.txt file to make sure that the setup.py file is up to date. Specifically that it has the install requirements it needs in it.